tests/simulator/pybind.cpp

```diff
@@ -0,0 +1,122 @@
+
```

Comment:
this file should be renamed it's not convenient to have multiple files with the same name and end up with tons of include.h and pybind.cpp

Reply:
currently there's only one pybind.cpp but point taken, I dislike redundancy tho